project/src/utils.js

```diff
@@ -13,5 +13,27 @@ const RatingToText = (rating) =>{
     return Ratings.AWESOME;
   }
 };
+const getTimeFormate = (num) => {
```

Comment:
символы не экономим

https://up.htmlacademy.ru/react/7/criteries#b6

```suggestion
const getTimeFormate = (number) => {
```